src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -15,24 +15,34 @@ export const TableToolbar = (props) => {
     isLoading,
     onImport,
     onExport,
+    onAddRow,
+    onDeleteRow,
     editMode,
     customToolbarActions,
     labels,
+    getSelectedRowsCount,
   } = props;
   const classes = useStyles();
 
   return (
-    <div className={classes.tableToolbar} style={isReady ? { borderBottom: 'none' } : null}>
+    <div className={classes.tableToolbar} style={isReady ? { borderBottom: 'none' } : null} data-cy="table-toolbar">
       <FullscreenButton
         isFullscreen={isFullscreen}
         toggleFullscreen={toggleFullscreen}
         disabled={!(isReady && !isLoading)}
-        label={labels.fullscreen ?? 'Fullscreen'}
+        label={labels.fullscreen}
       />
-      {onImport && (
-        <ImportButton onImport={onImport} disabled={!(editMode && !isLoading)} label={labels.import ?? 'Import'} />
+      {onImport && <ImportButton onImport={onImport} label={labels.import} disabled={!(editMode && !isLoading)} />}
+      {onExport && <ExportButton onExport={onExport} label={labels?.export ?? 'Export'} disabled={isLoading} />}
```

Comment:
Why only this button has a fallback value for label?

Reply:
Rebase mistake, fixed